.goreleaser.yml

```diff
@@ -0,0 +1,81 @@
+project_name: centrifugo
+release:
+  github:
+    owner: centrifugal
+    name: centrifugo
+  name_template: '{{.Tag}}'
+  # If set to true, will not auto-publish the release.
+  draft: true
+  # If set to true, will mark the release as not ready for production.
+  prerelease: true
+brew:
+  commit_author:
+    name: Alexander Emelin
+    email: frvzmb@gmail.com
+  install: bin.install "centrifugo"
+  skip_upload: true
+builds:
+- goos:
+  - linux
+  - darwin
+  - windows
+  goarch:
+  - amd64
+  - "386"
+  main: .
+  ldflags: -s -w -X main.VERSION={{.Version}}
+  binary: centrifugo
+  env:
+    # https://github.com/goreleaser/goreleaser/issues/225
+    - CGO_ENABLED=0
+archive:
+  name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ if .Arm
+    }}v{{ .Arm }}{{ end }}'
+  format: tar.gz
+  files:
+  - licence*
+  - LICENCE*
+  - license*
+  - LICENSE*
+  - readme*
+  - README*
+  - changelog*
+  - CHANGELOG*
+  format_overrides:
+    - goos: windows
+      format: zip
+nfpm:
+  name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ if .Arm
+    }}v{{ .Arm }}{{ end }}'
+  homepage: https://github.com/centrifugal/centrifugo
+  maintainer: Alexander Emelin <frvzmb@gmail.com>
+  description: Centrifugo - real-time websocket messaging server.
+  license: MIT
+  formats:
```

Comment:
let's only create binary releases at moment without `deb` and `rpm` packages. We already have separate step for `deb` and `rpm` uploaded on packagecloud in Travis CI.

Reply:
We still can add them later if needed

Reply:
Commented